source/com.google.mlkit/digital-ink-recognition/Additions/DigitalInkRecognitionRegistrar.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+
+namespace Google.MLKit.Vision.Digitalink.Recognition.Internal
+{
+    public partial class DigitalInkRecognitionRegistrar
+    {
+        global::System.Collections.Generic.IList<global::Firebase.Components.Component>? global::Firebase.Components.IComponentRegistrar.Components
```

Comment:
I will add comment why this was necessary. 

https://github.com/dotnet/android-libraries/pull/1267

Seems like XPath does not alter `api.xml`. The problem is with covariant return type (`managedReturn`)

I need to attempt few more tricks.